test/integration/model/findAll/order.test.js

```diff
@@ -22,7 +22,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           });
         });
 
-        if (current.dialect.name !== 'mssql') {
+        if (current.dialect.name !== 'mssql' && current.dialect.name !== 'oracle') {
```

Comment:
This should work, mssql should not have been allowed to disable this test

Reply:
Oracle DB doesn't support operator in an order by clause